test_bounce.py

```
@@ -20,3 +21,24 @@ def test_invalid_domain_redirection(client):
     response = client.get('/', headers={'Host': 'invalid_domain'})
     assert response.status_code == 301
     assert response.headers['Location'] == 'http://harperrules.com/'
+
+
+def test_missing_host_header(client):
```

Ruff

Missing return type annotation for public function `test_missing_host_header`

Add return type annotation: `None`

(ANN201)